2021-12-16 19:13:50.038503 | API is disabled on startup
2021-12-16 19:13:50.038503 | Settings loaded!
2021-12-16 19:13:51.548036 | Started VRCA loading process...
2021-12-16 19:13:51.549037 | Propting user to select VRCA...
2021-12-16 19:13:55.660536 | Loaded!
2021-12-16 19:13:56.534222 | Extract VRCA started...
2021-12-16 19:13:56.535223 | Deciding asset URL...
2021-12-16 19:13:56.546233 | Extracting loaded VRCA...
2021-12-16 19:13:56.547235 | Starting extract on new thread...
2021-12-16 19:13:56.548236 | In asset ripper folder
2021-12-16 19:13:56.548236 | Old rip found!
2021-12-16 19:13:56.601283 | Old rip deleted!
2021-12-16 19:13:56.601283 | Returned to base!
2021-12-16 19:13:56.602285 | In asset ripper folder!
2021-12-16 19:14:48.052875 | Extract complete! Cleaning...
2021-12-16 19:14:48.053877 | An error occured during extraction: Traceback (most recent call last):
  File "C:\Users\adamc\Documents\GitHub\A.R.E.S\ARES(rewritten)\ExternalFunctions.py", line 213, in ExtractVRCA
    os.rename("Shader",".Shader")
FileNotFoundError: [WinError 2] The system cannot find the file specified: 'Shader' -> '.Shader'

2021-12-16 19:14:48.056879 | An error occured while extracting a VRCA: Traceback (most recent call last):
  File "C:/Users/adamc/Documents/GitHub/A.R.E.S/ARES(rewritten)/GUI.py", line 348, in ExtractVRCAWrapper
    thread.join()
  File "C:\ProgramData\Anaconda3\envs\ARES(rewritten)\lib\threading.py", line 1011, in join
    self._wait_for_tstate_lock()
  File "C:\ProgramData\Anaconda3\envs\ARES(rewritten)\lib\threading.py", line 1027, in _wait_for_tstate_lock
    elif lock.acquire(block, timeout):
KeyboardInterrupt

